hugo/layouts/index.html

```diff
@@ -3,20 +3,34 @@
 	<p>Welcome to <b>official website of the Mumble project!</b></p>
 	<p><a class="download-button" href="{{ relref . "downloads" }}">Download now</a></p>
 
-	<h2>Latest News</h2>
-	<div>
-		{{ $blog := .Site.GetPage "blog" }}
-		{{ $pages := $blog.Pages }}
-		{{ $news := where $pages ".Params.categories" "intersect" (slice "News") }}
-		{{ $page := first 1 $news }}
-		{{ range $page }}
-			<div class="markdown-body">
-				<h3><a href="{{ .RelPermalink }}">{{ .Title }}</a></h3>
-				{{ partial "blog/teaser" . }}
+	{{ $blogpages := (.Site.GetPage "blog").Pages }}
+	{{ $news := where $blogpages ".Params.categories" "intersect" (slice "News") }}
+	{{ $releases := where $blogpages ".Params.categories" "intersect" (slice "Release") }}
+	<div id="home-newsblock">
+		<div>
+			<h2>Latest News</h2>
```

Comment:
```suggestion
			<h2>Latest news</h2>
```

Reply:
I think as a header I prefer the stylized capitalization of all words.

Reply:
Yeah, me too